app/main.py

```diff
@@ -1,5 +1,68 @@
-class Cargo:
-    def __init__(self, weight):
+class BaseRobot:
+    def __init__(self,
+                 name: str,
+                 weight: int,
+                 coords: list = None
+                 ) -> None:
+        self.name = name
         self.weight = weight
+        self.coords = [0, 0] if coords is None else coords
+
+    def go_forward(self, step: int = 1) -> None:
+        self.coords[1] += step
+
+    def go_back(self, step: int = 1) -> None:
+        self.coords[1] -= step
+
+    def go_right(self, step: int = 1) -> None:
+        self.coords[0] += step
+
+    def go_left(self, step: int = 1) -> None:
+        self.coords[0] -= step
+
+    def get_info(self) -> str:
+        return f"Robot: {self.name}, Weight: {self.weight}"
+
+
+class FlyingRobot(BaseRobot):
+    def __init__(self,
+                 name: str,
+                 weight: int,
+                 coords: list = None
+                 ) -> None:
+        super().__init__(name, weight, coords)
+
+        self.coords = [0, 0, 0] if coords is None else coords
 
-# write your code here
+    def go_up(self, step: int = 1) -> None:
+        self.coords[2] += step
+
+    def go_down(self, step: int = 1) -> None:
+        self.coords[2] -= step
+
+
+class DeliveryDrone(FlyingRobot):
+    def __init__(self,
+                 name: str,
+                 weight: int,
+                 max_load_weight: int,
+                 current_load: object,
+                 coords: list = None,
+                 ) -> None:
+        self.max_load_weight = max_load_weight
+        self.current_load = current_load
+        super().__init__(name, weight, coords)
+
+    def hook_load(self, cargo: object) -> None:
```

Comment:
```suggestion
    def hook_load(self, cargo: "Cargo") -> None:
```
you can use strings, if class is lower, or even better - just move that class higher